benchmarks/hash_anchored_vs_plain.py

```
@@ -0,0 +1,427 @@
+    doc1.replace("committee", "BOARD", paragraph=old_refs[5].split("#")[0] + "#" + _get_fresh_hash(doc1, 5))
+
+    # Now try to edit P6-P15 using the OLD occurrence indices
+    for para_idx in target_paras:
+        old_occ = target_occurrences.get(para_idx)
+        if old_occ is None:
+            plain_error += 1
+            continue
+
+        marker = f"[P{para_idx:02d}]"
+        try:
+            doc1.replace("committee", f"EDITED", occurrence=old_occ)
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)